include/klee/util/Bits.h

```diff
@@ -88,9 +95,11 @@ namespace klee {
     // @pre((x&(x-1)) == 0)
     // @post((1 << retval) == x)
     inline unsigned indexOfSingleBit(uint64_t x) {
+      assert((x & (x - 1)) == 0);
       unsigned res = bits32::indexOfSingleBit((unsigned) (x | (x>>32)));
-      if (x&((uint64_t) 0xFFFFFFFF << 32))
-	  res += 32;
+      if (x & ((uint64_t)0xFFFFFFFF << 32))
+        res += 32;
```

Comment:
You could although the current construction of that code guarantees that `res` is `<= 31`. I'll leave it up to you whether or not you want to add an assert there. If you do add it make sure it is before `res` is used to do a shift.